src/app/views/notifications/_notifications.html.erb

```diff
@@ -0,0 +1,3 @@
+<%= render notifications %>
+
+<%= link_to 'Load More', notifications_path(offset: @next), remote: true, class: 'button is-fullwidth more' if @next %>
```

Comment:
we didn't pass the `state` parameter here, so this link will return the next page for `all` notifications